app.py

```diff
@@ -0,0 +1,58 @@
+from flask import Flask, render_template, request, redirect
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from werkzeug import exceptions
+import random, string
+
+# Set up the app
+app = Flask(__name__)
+CORS(app)
+
+# Set up the database
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///urls.db"
```

Comment:
There's a deprecation warning as I run the server, I recommend adding this to the next line of code to remove this: app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False